app/views/starterDownload.scala.html

```diff
@@ -0,0 +1,35 @@
+@(downloads: PlayExamples)
+
+@renderProject(project: ExampleProject) = {
+<tr>
+  <td><strong><a href="@project.downloadUrl" target="_blank">@project.displayName</a></strong></td>
+  <td><a href="@project.downloadUrl" target="_blank">Download (zip)</a></td>
+  <td><a href="@project.gitHubUrl" target="_blank">View on GitHub</a></td>
+</tr>
+}
+
+<article class="starter">
+  <h2 id="starter">Play Starter Projects</h2>
```

Comment:
@wsargent Either this should be `id="starters"` or the above link should be `href="#starter"`. Right now the `href` doesn't work.